scripts/push-to-quay

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+echo -n "Relative path to your manifest file: "
+read FILE
```

Comment:
Please take this input as command line parameters. You should also add a help option.

Reply:
This can be done in a follow up PR